src/components/tooltip/tooltip-theme.scss

```diff
@@ -1,6 +1,6 @@
-md-tooltip.md-THEME_NAME-theme {
-  color: '{{background-700-contrast}}';
-  .md-content {
-    background-color: '{{background-700}}';
+.md-tooltip.md-THEME_NAME-theme {
+  color: '{{background-A100}}';
```

Comment:
@bradrich I don't know what you mean by "have the same color scheme", but the material design spec says `background-700`. https://material.google.com/components/tooltips.html#tooltips-tooltips-desktop-

By having these colors changed back, you'd be reintroducing old issues with dark themes and non-gray background palettes.